templates/prometheus.yaml

```diff
@@ -302,4 +302,4 @@ spec:
           # override the default k8s.gcr.io/kube-state-metrics repositry
           # containerd mirror functionality does not support pulling these images
           # TODO remove once https://github.com/containerd/containerd/issues/3756 is resolved
-          repository: gcr.io/google_containers/kube-state-metrics
+          repository: quay.io/coreos/kube-state-metrics
```

Comment:
Why has this changed? Sorry I don't have any context to review this PR so sorry for what could be a stupid question.

Reply:
that is the new image repository used by the official chart to use v1.8.0. That version isn't available for `gcr.io/google_containers/kube-state-metrics`

Reply:
I am being explicit here about this change